client/src/components/AddNewProfile.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import { Form } from "semantic-ui-react";
+import { getProfiles } from "../api/profiles";
+
+const AddNewProfile = () => {
+  const [profileData, setProfileData] = useState({
+    firstname: "",
+    lastname: "",
+    dob: "",
+    address: "",
+    email: "",
+    phone: "",
+    gender: "",
+    groups: "",
+    support_type: ""
+  });
+
+  const [profileCreated, setProfileCreated] = useState(null);
+
+  const handleChange = event => {
+    const updatedProfileData = {
+      ...profileData,
+      [event.target.name]: event.target.value
+    };
+
+    setProfileData(updatedProfileData);
+  };
+
+  const createProfile = event => {
+    getProfiles(profileData).then(isSuccessful => {
+      console.log(isSuccessful);
+      setProfileCreated(isSuccessful);
+      event.preventDefault();
+    });
+  };
+
+  return (
+    <Form onSubmit={createProfile}>
+      <Form.Field>
+        <label>First name</label>
```

Comment:
```suggestion
        <label for="first-name">First name</label>
```
Great job adding `id` attributes to the inputs! However, we still need to connect the inputs to the labels, and to do this we need to add a `for` attribute to all the labels with the same value as the corresponding input `id`. Let's update all the labels to have `for` attributes`